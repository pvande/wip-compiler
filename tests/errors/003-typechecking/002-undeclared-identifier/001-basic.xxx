// @TODO This outputs two errors, when one should be sufficient.
x := y
