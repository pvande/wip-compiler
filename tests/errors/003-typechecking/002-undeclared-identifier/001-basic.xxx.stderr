

Error: "Could not infer the type of this variable"
In [1;37mtests/errors/003-typechecking/002-undeclared-identifier/001-basic.xxx[0m on line [1;37m2[0m

> [0;36mx := y[0m
  [0;31m^     [0m
Error: "Declaration for that identifier was not found"
In [1;37mtests/errors/003-typechecking/002-undeclared-identifier/001-basic.xxx[0m on line [1;37m2[0m

> [0;36mx := y[0m
  [0;31m     ^[0m
